src/components/svgs/stations/gzmtr-int.tsx

```diff
@@ -180,9 +181,9 @@ const GzmtrIntStation = (props: StationComponentProps) => {
                 {transfer[0]?.map((info, i, arr) => (
                     <g
                         key={`gzmtr_int_${id}_stn_${i}`}
-                        transform={`translate(${CODE_POS[arr.length][i][0]},${CODE_POS[arr.length][i][1]})`}
+                        transform={`translate(${CODE_POS[arr.length][i][0]},${CODE_POS[arr.length][i][1]})scale(0.75)`}
                     >
-                        <StationNumber strokeColor={info[2]} lineCode={info[4]} stationCode={info[5]} />
+                        <StationNumber strokeColour={info[2]} lineNum={info[4]} stnNum={info[5]} />
```

Comment:
We use "color" all the time, but why change to "colour" here

Reply:
`StationNumber` from @railmapgen/svg-assets uses `strokeColour` as its parameter name :(

Reply:
Oh alright. 😉